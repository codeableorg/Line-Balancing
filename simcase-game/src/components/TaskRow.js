/** @jsx jsx */
import { jsx } from "@emotion/core";

const container = {
  background: "rgb(213,245,245)",
  borderRadius: "4px",
  display: "table",
  tableLayout: "fixed",
  width: "90%",
  height: "4vh",
  boxSizing: "border-box",
  marginLeft: "auto",
  marginRight: "auto",
  marginTop: "16px", //Needs a little fix
  marginBottom: "16px" // Must be automatic
};

const item = {
  display: "table-cell",
  verticalAlign: "middle",
  textAlign: "center",
  fontFamily: "Arial",
  color: "rgb(61,125,255)",
  fontSize: "0.9em"
};

function TitlesForm({ task = "Brainstorm (1.0h)" }) {
  return (
    <div css={container}>
      {/* Task name */}
      <div css={item}>{task}</div>

      {/* Radio buttons üçé*/}
      <div css={item}>
        <input type="radio" id="radio1" name="radio" checked />
      </div>
      <div css={item}>
        <input type="radio" id="radio2" name="radio" checked />
      </div>
      <div css={item}>
        <input type="radio" id="radio3" name="radio" checked />
      </div>
    </div>
  );
}

export default TitlesForm;
