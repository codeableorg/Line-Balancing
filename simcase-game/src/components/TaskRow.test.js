import React from "react";
import { render } from "@testing-library/react";

import TaskRow from "./TaskRow";

test("TaskRow based on game stage", () => {
  const { asFragment } = render(<TaskRow />);
  expect(asFragment()).toMatchSnapshot();
});
