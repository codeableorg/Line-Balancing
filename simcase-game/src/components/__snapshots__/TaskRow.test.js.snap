// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TaskRow based on game stage 1`] = `
<DocumentFragment>
  <div
    class="css-19ixip1-TitlesForm"
  >
    <div
      class="css-1qrkjvu-TitlesForm"
    />
    <div
      class="css-1qrkjvu-TitlesForm"
    >
      <input
        id="radio1"
        type="radio"
      />
    </div>
    <div
      class="css-1qrkjvu-TitlesForm"
    >
      <input
        id="radio2"
        type="radio"
      />
    </div>
    <div
      class="css-1qrkjvu-TitlesForm"
    >
      <input
        id="radio3"
        type="radio"
      />
    </div>
  </div>
</DocumentFragment>
`;
